src/App.js

```diff
@@ -11,6 +12,7 @@ const App = () => {
       <Stack.Navigator>
         <Stack.Screen name="Home" component={HomeScreen} />
         <Stack.Screen name="SignUp" component={SignUpScreen} />
+        <Stack.Screen name="Profile" component={ProfileScreen} />
```

Comment:
After merging with #23 add page header title to it

Reply:
solve in d64f27a4351abcc9d759dbb9d55e3b8f238d6006